build.cmd

```diff
@@ -204,6 +204,19 @@ call                                 "%__VSToolsRoot%\VsDevCmd.bat"
 
 REM =========================================================================================
 REM ===
+REM === Restore optimization profile data
+REM ===
+REM =========================================================================================
+
+echo %__MsgPrefix%Restoring the OptimizationData Package
```

Comment:
Fair enough--I was trying to keep from adding complexity to the usage (having to know the extra command to skip), but you raise a good point about new platform bring-up. I'll just add a skip argument then, and keep things simple.